src/mumble/MainWindow.cpp

```diff
@@ -428,6 +428,12 @@ void MainWindow::createActions() {
 	gsMoveBack = new GlobalShortcut(this, GlobalShortcutType::MoveBack, tr("Move back", "Global shortcut"));
 	gsMoveBack->setObjectName("gsMoveBack");
 	gsMoveBack->qsWhatsThis = tr("This will move you back into your previous channel");
+
+	gsAdaptivePush = new GlobalShortcut(this, GlobalShortcutType::AdaptivePush, tr("Adaptive Push", "Global Shortcut"));
+	gsAdaptivePush->setObjectName(QLatin1String("gsAdaptivePush"));
+	gsAdaptivePush->qsToolTip =
+		tr("This will act as push-to-talk when using push-to-talk transmission, and push-to-mute otherwise",
```

Comment:
```suggestion
		tr("When using the push-to-talk transmission mode, this will act as the push-to-talk action. Otherwise, it will act as a push-to-mute action. ",
```